XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml

```diff
@@ -31,15 +31,15 @@
         </DataTemplate>
 
         <DataTemplate x:Key="NormalItemTemplate" x:DataType="x:Int32">
-            <Button HorizontalAlignment="Stretch" VerticalAlignment="Stretch" Background="{ThemeResource SystemChromeLowColor}">
-                <TextBlock Text="{x:Bind}"/>
-            </Button>
+            <Border Background="{ThemeResource SystemControlBackgroundChromeMediumBrush}">
+                <TextBlock Text="{x:Bind}" HorizontalAlignment="Center" VerticalAlignment="Center" />
+            </Border>
         </DataTemplate>
 
         <DataTemplate x:Key="AccentItemTemplate" x:DataType="x:Int32">
-            <Button HorizontalAlignment="Stretch" VerticalAlignment="Stretch" Background="{ThemeResource SystemAccentColor}">
-                <TextBlock Text="{x:Bind}"/>
-            </Button>
+            <Border  Background="{ThemeResource SystemControlBackgroundAccentBrush}" >
+                <TextBlock Text="{x:Bind}"  HorizontalAlignment="Center" VerticalAlignment="Center" Foreground="{ThemeResource SystemControlForegroundChromeWhiteBrush}"/>
```

Comment:
Does the chrome white brush here work with both light/dark theme?

Reply:
I realized the brushes used in XCG are all pointing towards older brushes. I'll make a note go through them and update them to the new brushes recommended by design in the visual updates.

Reply:
Here is what it looks like in dark theme. A border might help in high contrast theme.
![image](https://user-images.githubusercontent.com/28935693/162042374-764c5230-9b01-42e1-a902-0acee4dccd29.png)